.gitignore

```diff
@@ -136,4 +136,6 @@ dmypy.json
 [Ll]ocal
 [Ss]cripts
 pyvenv.cfg
-pip-selfcheck.json
\ No newline at end of file
+pip-selfcheck.json
+
+.vscode
```

Comment:
Please don't add IDE specific stuff to gitignore files. It's a much better idea to use the global gitignore on your system :)